package.json

```diff
@@ -7,7 +7,9 @@
     "test": "test"
   },
   "scripts": {
-    "test": "./test/runtests.sh"
+    "test": "./test/runtests.sh",
+    "compile": "rm -rf lib/ && babel --source-maps -d lib/ src/",
```

Comment:
Thinking about this I suspect tc-base could define binaries that wrap linter, mocha, babel, etc... Then we just call them through the wrapped, and ideally it just has 2 wrappers: run-tests and compile.

Then this tc-base component dictates where code lives, but who cares if it's a separate module...